js/component/DisableForms.js

```diff
@@ -13,10 +13,10 @@ export default function () {
 
                 if (! el.el.contains(node)) return
 
-                if (node.tagName.toLowerCase() === 'button' && node.type === 'submit') {
+                if ((node.tagName.toLowerCase() === 'button' && node.type === 'submit') || node.tagName.toLowerCase() === 'select') {
```

Comment:
Does this prevent the value from being sent with the request? Or is the data already on the way before the handler is called? I know if a `select` is disabled when a form is submitted, the value doesn't get included.

Reply:
I thought about the same thing before doing this PR, but in case of Livewire, the default behavior is the fields with wire:model and on wire:submit, all fields values will be already in their respective public properties.

Reply:
Good to know, thank you!